src/test/ui/impl-trait/bound-normalization.rs

```diff
@@ -0,0 +1,102 @@
+// compile-fail
+// edition:2018
+
+#![feature(async_await)]
+#![feature(existential_type)]
+#![feature(impl_trait_in_bindings)]
+//~^ WARNING the feature `impl_trait_in_bindings` is incomplete
+
+// See issue 60414
+
+/////////////////////////////////////////////
+// Original report
```

Comment:
E.g. because this section seems to be `// check-pass` and not `// compile-fail`.